integration/raft/channel_participation_test.go

```diff
@@ -303,3 +255,85 @@ func applicationChannelGenesisBlock(n *nwo.Network, o *nwo.Orderer, p *nwo.Peer,
 
 	return genesisBlock
 }
+
+func ordererOrganizationsAndConsenters(n *nwo.Network, orderers []*nwo.Orderer) ([]configtx.Organization, []orderer.Consenter) {
+	ordererOrgsMap := map[string]*configtx.Organization{}
+	consenters := make([]orderer.Consenter, len(orderers))
+
+	for i, o := range orderers {
+		rootCert := nwo.ParseCertificate(n.OrdererCACert(o))
+		adminCert := nwo.ParseCertificate(n.OrdererUserCert(o, "Admin"))
+		tlsRootCert := nwo.ParseCertificate(filepath.Join(n.OrdererLocalTLSDir(o), "ca.crt"))
```

Comment:
@wlahti I moved this out of the `nwo` package earlier today. There should be an exported version in this suite.

You'll need to rebase on master to avoid a build break post-merge.

Reply:
Thanks for the heads up. Done.